src/robot/libdocpkg/htmlwriter.py

```diff
@@ -48,39 +48,6 @@ def write_data(self):
         JsonWriter(self._output).write_json('libdoc = ', self._libdoc)
 
 
-class JsonConverter(object):
-
-    def __init__(self, doc_formatter):
-        self._doc_formatter = doc_formatter
-
-    def convert(self, libdoc):
-        return {
-            'name': libdoc.name,
-            'doc': self._doc_formatter.html(libdoc.doc, intro=True),
-            'version': libdoc.version,
-            'named_args': libdoc.named_args,
-            'scope': libdoc.scope,
-            'generated': get_timestamp(daysep='-', millissep=None),
-            'inits': self._get_keywords(libdoc.inits),
-            'keywords': self._get_keywords(libdoc.keywords),
-            'all_tags': tuple(libdoc.all_tags),
-            'contains_tags': bool(libdoc.all_tags)
-        }
-
-    def _get_keywords(self, keywords):
-        return [self._convert_keyword(kw) for kw in keywords]
-
-    def _convert_keyword(self, kw):
-        return {
-            'name': kw.name,
-            'args': kw.args,
-            'doc': self._doc_formatter.html(kw.doc),
-            'shortdoc': ' '.join(kw.shortdoc.splitlines()),
-            'tags': tuple(kw.tags),
-            'matched': True
-        }
-
-
```

Comment:
this "external" converter for the model is now included in the model.
Closer relation between classes and its dictionary conversion.